### PR TITLE
Version control git

### PR DESCRIPTION
**Pull request by liushilongbuaa**
**- Why I did it**
bug fix for reproducible build of git
**- How I did it**
What I fixed is the commit "bac9309". git used unexpected env REAL_COMMAND.
**- How to verify it**

**- Which release branch to backport (provide reason below if selected)**

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by liushilongbuaa** [comment]
retest this please

**Comment by liushilongbuaa** [comment]
"/AzurePipelines run"

File: Makefile.work [hunk 1]
```diff
@@ -114,7 +114,7 @@ $(shell CONFIGURED_ARCH=$(CONFIGURED_ARCH) j2 $(SLAVE_DIR)/Dockerfile.user.j2 >
 $(shell BUILD_SLAVE=y scripts/prepare_docker_buildinfo.sh $(SLAVE_BASE_IMAGE) $(SLAVE_DIR)/Dockerfile $(CONFIGURED_ARCH) "" $(BLDENV))
 
 # Add the versions in the tag, if the version change, need to rebuild the slave
-SLAVE_BASE_TAG = $(shell cat $(SLAVE_DIR)/Dockerfile $(SLAVE_DIR)/buildinfo/versions/versions-* | sha1sum | awk '{print substr($$1,0,11);}')
+SLAVE_BASE_TAG = $(shell cat $(SLAVE_DIR)/Dockerfile $(SLAVE_DIR)/buildinfo/versions/versions-* src/sonic-build-hooks/hooks/* | sha1sum | awk '{print substr($$1,0,11);}')
```
Comment (xumia) [inline Makefile.work:117]: It is not necessary to add it, because the package will be re-installed in the salve containers. Please help check it.

**Review by xumia — APPROVED** [review]

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**